get-started/evaluate-elastic.md

```diff
@@ -0,0 +1,349 @@
+---
+products:
+  - id: elasticsearch
+  - id: elastic-stack
+  - id: observability
+  - id: security
+applies_to:
+  serverless:
+  deployment:
+    ess: ga
+description: Build a successful proof of concept during your Elastic trial. Learn how to define success criteria, choose the right deployment and use case, measure results, and prepare for production.
+---
+
+# Evaluate Elastic during a trial
+
+If you're evaluating Elastic during a trial, this guide helps you build a meaningful proof of concept (PoC) that demonstrates clear value to your organization.
+
+Rather than prescribing specific technical steps, this guide focuses on the evaluation process itself, helping you make strategic decisions and measure success.
+
+For an overview, check out [Explore Elastic Cloud with your 14‑day free trial](https://www.elastic.co/cloud/cloud-trial-overview).
+
+## What's included in your trial
+
+Your Elastic trial gives you full access to explore our platform's capabilities:
+
+- All features available across [Search](/solutions/search.md), [{{observability}}](/solutions/observability.md), and [Security](/solutions/security.md) solutions, depending on your choice of deployment and project type.
+- A choice between {{serverless-full}} and {{ech}} deployment types.
+- Access to integrations, {{ml-features}}, and advanced analytics.
```

Comment:
```suggestion
- Integrations, {{ml-features}}, and advanced analytics.
```

Reply:
Since I am evaluating Elastic, why do I care about integrations, machine learning, and advanced analytics?